backend/backend_test_data.json

```diff
@@ -130,7 +130,18 @@
       "limitPerPlayer": 2
     }
   },
-  "notifications": {},
+  "notifications": {
+    "notification-$(ident)s-abc": {
+        "groupId": "group-%(ident)s",
+        "message": "hello",
+        "app": true,
+        "vibrate": false,
+        "sound": "none",
+        "destination": "https://google.com",
+        "sendTime": 1494381500,
```

Comment:
This is going to foul up tests quickly. See the clean data method in the test file to handle this.